frontend/src/pages/ProjectDetails/index.jsx

```diff
@@ -0,0 +1,102 @@
+import React, {} from "react";
+
+import { Col, Tag, Avatar } from "antd";
+import { UnlockFilled, PlusOutlined, FileOutlined} from "@ant-design/icons";
+import Frame from "../../components/Frame";
+import {
+  H1,H2,H3,H4,
+  Main,
+  Date,
+  Tags,
+  Summary,
+  Deadlines,
+  Files,
+  FileContainer,
+  FileDiv,
+  Side,
+  UserDiv,
+  FadedText,
+} from "./style";
+
+const ProjectDetails = () => {
+  const data = {
+    title: "Logistic Regression for Bicycle Manufacturers",
+    tags: ["Mechanics", "Computer Science", "Statistics", "Artificial Intelligence", "Machine Learning"],
+    summary: "The purpose of this article is to provide researchers, editors, and readers with a set of guidelines for what to expect in an article using logistic regression techniques. Tables, figures, and charts that should be included to comprehensively assess the results and assumptions to be verified are discussed.",
+    deadlines: [{"date": "04 March, 2021", "desc": "Baseline results"}, {"date": "24 December, 2021", "desc": "Application Deadline for IOCON 2021"}],
+    duedate: "Due 06 July, 2021",
+    requirements: "Here is the requirements.",
```

Comment:
Have we checked with the backend team as to what kind of a new structure they are going to implement for the requirements after the milestone1 feedback?

Reply:
That haven't discussed yet. These are just mock data / data structure to develop initial version of the project display page. It might change in future.